src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/CapturePlugin.php

```diff
@@ -22,15 +22,14 @@ class CapturePlugin extends AbstractComputopPlugin implements CommandByOrderInte
      *
      * @api
      *
-     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem[] $orderItems
+     * @param array<\Orm\Zed\Sales\Persistence\SpySalesOrderItem> $orderItems
      * @param \Orm\Zed\Sales\Persistence\SpySalesOrder $orderEntity
      * @param \Spryker\Zed\Oms\Business\Util\ReadOnlyArrayObject $data
      *
      * @return \Spryker\Shared\Kernel\Transfer\TransferInterface|array
      */
     public function run(array $orderItems, SpySalesOrder $orderEntity, ReadOnlyArrayObject $data)
     {
-        $orderEntity->getItems()->setData($orderItems);
```

Comment:
why was it removed?
looks like API behavior change

Reply:
Fixed